src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -72,35 +74,73 @@ To set up the IAM role:
 
 When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
 
+
+## Set up an Azure Event Hub [#event-hub-setup]
+
+To set up streaming data export to Azure, you must first set up an Event Hub. We'll walk you through that procedure in the next three steps.
+
+Alternatively, you can follow the Azure guide [here](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+
+### Step 1. Create an Event Hubs namespace [#create-event-hubs-namespace]
+
+1. Navigate to Event Hubs within your Microsoft Azure account.
+2. Follow the steps to create an Event Hubs namespace. We recommend enabling auto-inflate to ensure you receive all of your data.
+3. Ensure public access is enabled, we will be using a Shared Access Policy to authenticate securely with your Event Hub.
+4. Once your Event Hubs namespace is deployed, click **Go to resource**.
+
+### Step 2. Create an Event Hub[#create-event-hub]
+
+1. In the left column, click **Event Hubs**.
+2. Then click **+Event Hub** to create an Event Hub.
+2. Enter the desired Event Hub Name. Save this, as you'll need it later to create the streaming export rule.
+4. Once the Event Hub is created, click on the Event Hub.
+
+### Step 3. Create and attach a shared access policy [#event-hub-policy]
+
+1. In the left column, go to **Shared access policies**.
+2. Click **+Add** near the top of the page.
+3. Choose a name for your shared access policy.
+4. Check **Send**, and click **Create**.
+5. Click the created policy, and copy the **Connection string–primary key**. Save this, as we'll use this to authenticate and send data to your Event Hub.
+
+When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
```

Comment:
Small thought, would it be better to link to the next most relevant section for them to read? It may be the next one but it seems kind of nebulous in a document like this. Again, just a small thing to think about.